packages/mrm/src/__tests__/index.spec.js

```diff
@@ -106,6 +123,61 @@ describe('tryResolve', () => {
 	});
 });
 
+describe('installGlobalPackage', () => {
+	it('should resolve to true if able to install the package', async () => {
+		spawn.mockReturnValueOnce({ on: spawnOnErrorMock });
+		const pkgName = String(Date.now());
```

Comment:
I'd use fixed strings instead of dates, this makes the test code harder to unserstand.